### ISSUE TITLE
Typing error with 2.0.9

### ISSUE THREAD

**Issue by Jogai** [primefaces/primevue#474]
Upgrade to 2.0.9 gives me an error about typings. I do have the vue shim typings file like https://github.com/primefaces/primevue-typescript-quickstart

```
ERROR in C:/SRC/clientapp/src/views/Variants.vue(169,10):
169:10 Property '$toast' does not exist on type 'Admin'.
    167 |                               this.selectedItems = [];
    168 |                               this.load();
  > 169 |                               this.$toast.add({ severity: "success", summary: "Successful", detail: `${item.name} Deleted`, life: 3000 });
        |                                    ^

**Comment by cagataycivici** [comment]
You are using Vue 2 right?

**Comment by Jogai** [comment]
yes

**Comment by cagataycivici** [comment]
Unable to replicate with 2.4.1, please try with it and create a new ticket for review if issue persists.

**Comment by Jogai** [comment]
Sorry, I didnt track this actively. I think it resolved in the 2.1.x versions. 2.4.1 is working correct for me too
